docs/content/en/docs/install/install.md

```diff
@@ -80,44 +74,79 @@ as well as for upgrades.
 Some helpful hints:
 
 * Use the `--version <version>` flag on the
-  `helm upgrade --install` command line to specify a different Keptn version.
+  `helm upgrade --install` command to install a Keptn version
+  other than the latest.
+  For example,
+
+  ```shell
+  helm upgrade --install keptn keptn/keptn \
+   --version version klt-v0.8.2 \
+   -n keptn-lifecycle-toolkit-system --create-namespace --wait
+  ```
 
-* Use the following command sequence to see a list of available versions:
+  * Use the following command sequence to see a list of available versions:
+
+    ```shell
+    helm repo update
+    helm search repo keptn
+    ```
+
+* If you modify the `values.yaml` file for the umbrella chart
+  and/or the component sub-charts,
+  apply the modified configuration with the `values - values.yaml` flag.
```

Comment:
```suggestion
  apply the modified configuration with the `--values <values-filename>` flag.
```

Reply:
Whoops, I had done some of this manually so i implemented the rest manually.  Done.